src/components/Text/Text.styles.ts

```diff
@@ -1,139 +1,121 @@
+    letter-spacing: ${cVar('typographyDesktopH800LetterSpacing')};
+    text-transform: ${cVar('typographyDesktopH800TextTransform')};
   `,
-  h2: styled.h2<TextProps>`
+  h700: styled.h2<TextProps>`
     ${baseStyles};
 
-    font-size: ${oldTypography.sizes.h2};
-    line-height: ${oldTypography.lineHeights.h2};
-    font-weight: ${oldTypography.weights.bold};
-    font-family: ${oldTypography.fonts.headers};
+    font: ${cVar('typographyDesktopH700')};
+    letter-spacing: ${cVar('typographyDesktopH700LetterSpacing')};
+    text-transform: ${cVar('typographyDesktopH700TextTransform')};
   `,
-  h3: styled.h3<TextProps>`
+  h600: styled.h3<TextProps>`
     ${baseStyles};
 
-    font-size: ${oldTypography.sizes.h3};
-    line-height: ${oldTypography.lineHeights.h3};
-    font-weight: ${oldTypography.weights.bold};
-    font-family: ${oldTypography.fonts.headers};
+    font: ${cVar('typographyDesktopH600')};
+    letter-spacing: ${cVar('typographyDesktopH600LetterSpacing')};
+    text-transform: ${cVar('typographyDesktopH600TextTransform')};
   `,
-  h4: styled.h4<TextProps>`
+  h500: styled.h4<TextProps>`
     ${baseStyles};
 
-    font-size: ${oldTypography.sizes.h4};
-    line-height: ${oldTypography.lineHeights.h4};
-    font-weight: ${oldTypography.weights.bold};
-    font-family: ${oldTypography.fonts.headers};
+    font: ${cVar('typographyDesktopH500')};
+    letter-spacing: ${cVar('typographyDesktopH500LetterSpacing')};
+    text-transform: ${cVar('typographyDesktopH500TextTransform')};
   `,
-  h5: styled.h5<TextProps>`
+  h400: styled.h5<TextProps>`
     ${baseStyles};
 
-    font-size: ${oldTypography.sizes.h5};
-    line-height: ${oldTypography.lineHeights.h5};
-    font-weight: ${oldTypography.weights.bold};
-    font-family: ${oldTypography.fonts.headers};
+    font: ${cVar('typographyDesktopH400')};
+    letter-spacing: ${cVar('typographyDesktopH400LetterSpacing')};
+    text-transform: ${cVar('typographyDesktopH400TextTransform')};
   `,
-  h6: styled.h6<TextProps>`
+  h300: styled.h6<TextProps>`
     ${baseStyles};
 
-    font-size: ${oldTypography.sizes.h6};
-    line-height: ${oldTypography.lineHeights.h6};
-    font-weight: ${oldTypography.weights.bold};
-    font-family: ${oldTypography.fonts.headers};
+    font: ${cVar('typographyDesktopH300')};
+    letter-spacing: ${cVar('typographyDesktopH300LetterSpacing')};
+    text-transform: ${cVar('typographyDesktopH300TextTransform')};
   `,
-  subtitle1: styled.span<TextProps>`
+  h200: styled.span<TextProps>`
     ${baseStyles};
 
-    font-size: ${oldTypography.sizes.subtitle1};
-    line-height: ${oldTypography.lineHeights.subtitle1};
-    font-weight: ${oldTypography.weights.regular};
+    font: ${cVar('typographyDesktopH200')};
+    letter-spacing: ${cVar('typographyDesktopH200LetterSpacing')};
+    text-transform: ${cVar('typographyDesktopH200TextTransform')};
   `,
-  subtitle2: styled.span<TextProps>`
+  h100: styled.span<TextProps>`
```

Comment:
All the header styles render as `h*` but these 2 as `span`s, let's use `h6` for those